src/pubtools/sign/signers/cosignsigner.py

```diff
@@ -316,7 +318,12 @@ def existing_signatures(self, reference: str) -> Tuple[bool, str]:
             return True, ""
 
 
-def cosign_container_sign(signing_key=None, config="", digest=None, reference=None):
+def cosign_container_sign(
+    signing_key: str | None = None,
```

Comment:
Default values here should be changed to "" instead of None for signing_key and config. For digest and reference to []